docs/src/advanced/arguments.md

```diff
@@ -16,6 +16,8 @@ Discordeno comes with the most useful command arguments already built for you. T
 - `member` When you want a member object and the user can provide a member id, @mention or their username/nickname.
 - `number` When you want a number.
 - `role` When you want a role object. The user can provide the role id, mention or role name.
+- `snowflake` When you just want a snowflake.
```

Comment:
You should maybe explain what a snowflake is 🤔  not everyone may know thats a unique generated id for users, channels, roles, guilds emojis etc